Assets/Scripts/Items/ItemPanel.cs

```diff
@@ -258,31 +210,104 @@ private void addItemToList(Item item)
 
     private void RegisterItem1Click(ClickEvent click)
     {
-        itemController.ItemPicked(selectedItems[0].id); //activate the item selected's code
-        itemChosen = true; 
-        addItemToList(selectedItems[0]);
-        selectedItems.Clear();
+        selectedIndex = 0;
+        selectedItemLabel.text = selectedItems[0].name;
+        confirmPanel.style.display = DisplayStyle.Flex;
     }
     private void RegisterItem2Click(ClickEvent click)
     {
-        itemController.ItemPicked(selectedItems[1].id); //activate the item selected's code
-        itemChosen = true; 
-        addItemToList(selectedItems[1]);
-        selectedItems.Clear();
+        selectedIndex = 1;
+        selectedItemLabel.text = selectedItems[1].name;
+        confirmPanel.style.display = DisplayStyle.Flex;
     }
     private void RegisterItem3Click(ClickEvent click)
     {
-        itemController.ItemPicked(selectedItems[2].id); //activate the item selected's code
-        itemChosen = true; 
-        addItemToList(selectedItems[2]);
-        selectedItems.Clear();
-
+        selectedIndex = 2;
+        selectedItemLabel.text = selectedItems[2].name;
+        confirmPanel.style.display = DisplayStyle.Flex;
     }
     private void RegisterSkipClick(ClickEvent click)
     {
-        itemController.ItemPicked(-1);
-        itemChosen = true; 
+        selectedIndex = -1;
+        selectedItemLabel.text = "Skip";
+        confirmPanel.style.display = DisplayStyle.Flex;
+    }
+
+    private void ConfirmSelection(ClickEvent click)
+    {
+        if (selectedIndex == -1)
+        {
+            PlayerStats.Instance.CurrentHealth += 5;
```

Comment:
I got some player testing feedback, they said perhaps changing this to 50% health back for sacrifice of an item. Something we discuss?

Reply:
Fixed